lib/pages/timetable/view/events/add_event_view.dart

```diff
@@ -273,6 +286,65 @@ class _AddEventViewState extends State<AddEventView> {
                               return null;
                             },
                           ),
+                        if ([UniEventType.lecture].contains(selectedEventType))
+                          Autocomplete<Person>(
```

Comment:
Can you make this whole thing a method or a separate widget? It's pretty large and makes the code hard to follow.